### DISCUSSION TITLE
which terminal are they pasting the signing key inside the security onion ?

### DISCUSSION THREAD

**Discussion by aleeshamichael** [Security-Onion-Solutions/securityonion#8480] [General] [answered]
Which terminal are they pasting the signing key inside the security onion ?

**Comment by dougburks** [comment]
I'm not sure that I understand your question. 

Who are you referring to when you say `they`?

Are you perhaps referring to the signing key referenced at https://securityonion.net/download? If so, you should be able to follow those instructions at any standard Linux terminal.

**Comment by aleeshamichael** [comment, marked as answer]
Hello I have a question, I tried to install security Onion bhy downloading the ISo image and started installing it in a VMware 16 pro , I set it up successfully , But I could nt connect my ip address inside the chrome , it says "the site can't be reached. The ip address tooks too long to respond. I tried to fix the issue, but it didnt help.

**Reply by dougburks** [reply]
Did you allow your IP address to connect using so-allow?
https://docs.securityonion.net/en/2.3/so-allow.html#so-allow

If you need further help, then please provide further information as described at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:
- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.140
- Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, did you install on CentOS 7 or Ubuntu?
- How many nodes do you have?
- What are the hardware specs of each of those nodes?
- How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.

**Reply by aleeshamichael** [reply]
Do you install from our Security Onion ISO image or did you perform a network installation?
I downloaded the ISO image.
Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
I chose standard over airgap.
I chose the latest version of security onion.
I didnt particularly didnt give any commands like sudo salt-call state.highstate, or so-status.

1.The_ guest operating system I chose is linux, version , i chose is centos 7 64 bit.
2.The maximum disk size used is 200gb, and I stored virtual disk as a single file.
3.I used 4 gb memory for virtual machines.
4.The no of processor core I used is 4.
5.The network adapter selected is NAT.
6.I used basic graphic mode.
7.I chose to install standalone production and I agreed and accepted the elastic license.
8.I got a message, the machine currently has 1 nic, but needs 2 to meet minimum requirements. Press yes to continue anyway.
9.Also got a message, the machine currently has 4 gb memory, but needs 12 gb to meet requirements. Press yes to continue anyway, or press no.
10.Enter a short description for the node or press enter to leave blank, since it is optional , I skipped.
11.Please select your management NIC: I selected ens33 00:0c:29:5f:33:0f link up.
12.Choose how to set up your management interface :Chose static ipv4 addresses.
13.Choose which type of manager to install?-I chose basic.
14.what tool would you like to generate metadata.-Zeek
15.Which Ids rule set would you like to use?-I chose ETOPEN
16.Do you want to keep the default docker, ip range. If you are unsure, please accept the default option of yes.I chose yes.
17.Would you like to configure an ntp servers?no
I was almost stuck in the "applying so-fleet setup salt state for 2 hours. The percentage of completion was 78% for 2 hours.

**Reply by dougburks** [reply]
> 3.I used 4 gb memory for virtual machines.

> 9.Also got a message, the machine currently has 4 gb memory, but needs 12 gb to meet requirements. Press yes to continue anyway, or press no.

If you can only allocate 4GB RAM to Security Onion, then you can only do an Import installation. For more information, please see:
https://docs.securityonion.net/en/2.3/first-time-users.html
https://docs.securityonion.net/en/2.3/hardware.html
